models/prices/ethereum/prices_ethereum_tokens.sql

```diff
@@ -1708,7 +1708,7 @@ FROM
     ('strk-starknet', 'ethereum', 'STRK',0xCa14007Eff0dB1f8135f4C25B34De49AB0d42766,18),
     ('ela-elastos', 'ethereum', 'ELA', 0xe6fd75ff38adca4b97fbcd938c86b98772431867,18),
     ('frxeth-frax-ether', 'ethereum', 'FRXETH',0x5e8422345238f34275888049021821e8e08caa1f,18),  
-    --('cgpt-chaingpt', 'ethereum', 'CGPT',0x25931894a86d47441213199621f1f2994e1c39aa,18), --not in API 
+    ('cgpt-chaingpt', 'ethereum', 'CGPT',0x25931894a86d47441213199621f1f2994e1c39aa,18), 
```

Comment:
looks like this is still failing due to missing id value?
see CI logs for details. must be why it was commented out